tls/vibe/stream/openssl.d

```diff
@@ -209,6 +209,23 @@ static if (!OPENSSL_VERSION.startsWith("1.0.") && !OPENSSL_VERSION.startsWith("0
 
 	// OpenSSL 1.1 renamed `sk_*` to OpenSSL_sk_*`
 	private alias OPENSSL_sk_free = sk_free;
+
+	// Temporary hack: Deimos OpenSSL v3.0.1 is missing bindings for OpenSSL v1.0.x
+	// Until it's updated, we have duplicates here, see:
+	// https://github.com/vibe-d/vibe.d/pull/2658
+	// https://github.com/vibe-d/vibe.d/pull/2661
+	extern(C) const(SSL_METHOD)* SSLv23_client_method();
+	extern(C) const(SSL_METHOD)* SSLv23_server_method();
+
+	extern(C) int CRYPTO_num_locks();
+	extern(C) void CRYPTO_set_locking_callback(
```

Comment:
Should those be public tho? If someone relies on `SSLv23_client_method` being "automatically" imported, then this needs to be deprecated afterwards. I don't have context on whether the openssl bindings are publicly imported.

Reply:
Note that I'm working on moving everything to the Deimos bindings so this should (eventually) go.